static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
Interesting problem - this is an action type of `NEW_AD_HEIGHT`? Maybe you could use `fastdom.defer` to push the css change to the next frame? Why do you need to remove it one animation frame after the animation end rather than in the same frame?

With the banner element, I think `will-change: height` is what I was implying, though perf optimisation wise, I don't know what the browser would do, maybe the `will-change` helps the browser to understand that sibling or parent re-paints are on their way or defer actions that would be overridden by the height change. I need to do a @regiskuckaertz and hunt down some source code to tell me...